<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_v</class>
 <widget class="QDialog" name="Dialog_v">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Des Visiteurs</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 39, 39);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>1001</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 0, 61);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableView" name="tab_visiteurs_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>240</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	
background-color: rgb(45, 45, 45);
	border-radius: 20px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pb_excel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 165, 21);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>EXCEL</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>799</x>
      <y>30</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	
background-color: rgb(45, 45, 45);
	border-radius: 20px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pb_tri">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 165, 21);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTri">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>numero_visiteur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prenom</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	
background-color: rgb(45, 45, 45);
	border-radius: 20px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	
background-color: rgb(45, 45, 45);
	border-radius: 20px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Numero</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_numero_supprimer_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_supprimer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(204, 0, 61);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	
background-color: rgb(45, 45, 45);
	border-radius: 20px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Numero</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_numero_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nom_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_prenom_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_tel_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Num_Tel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_adresse_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_age_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajouter_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(204, 0, 61);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>460</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 165, 21);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	
background-color: rgb(45, 45, 45);
	border-radius: 20px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Numero</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_numero_modifier_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_tester_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 165, 21);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Tester</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nom_modifier_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_prenom_modifier_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Num_Tel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_tel_modifier_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_adresse_modifier_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_modifier_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(204, 0, 61);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_age_modifier_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: rgb(255, 255, 255);

border-radius: 5px;

border: 2px solid rgb(194, 194, 194);

padding-left: 10px;

}

QLineEdit:hover {

border: 2px solid rgb(172, 172, 172);

}

QLineEdit:focus {

border: 2px solid rgb(172, 172, 172);

}

</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Quitter">
   <property name="geometry">
    <rect>
     <x>1108</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border-radius:20px;
background-color: rgb(56, 56, 56);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
color: rgb(56, 56, 56);
background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Smart Parc</string>
   </property>
  </widget>
  <widget class="QLabel" name="VV">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>VV.jpg</pixmap>
   </property>
  </widget>
  <zorder>VV</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_21</zorder>
  <zorder>groupBox_8</zorder>
  <zorder>Quitter</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
